config/dev.exs

```diff
@@ -0,0 +1,14 @@
+use Mix.Config
+
+config :boreale, Boreale,
+  port: String.to_integer(System.get_env("PORT") || "4000"),
+  page_title: System.get_env("PAGE_TITLE") || "Boréale Authentication"
+
+config :boreale, Boreale.Router,
+  secret_key_base:
+    System.get_env("SECRET_KEY_BASE") ||
+      "C6lIRaMxBMHQxo3fpdTuFJpYRC1Sa1JPWcWbISYzV5PdgIdMFQg/9L1ih1YhTPAo",
```

Comment:
These are dummy values, this is why I added them to a dev env file that is not loaded in prod. To prevent users from having those as default in prod